pyanaconda/packaging/__init__.py

```diff
@@ -230,7 +230,7 @@ def _sourceNeedsNetwork(self, sources):
         """
         network_protocols = ["http:", "ftp:", "nfs:", "nfsiso:"]
         for s in sources:
-            if any(s.startswith(p) for p in network_protocols):
+            if any(s.startswith(p) for p in network_protocols if s):
```

Comment:
Seems quite strange to me, I thing simple

if s and any(s.startswith(p) for p in network_protocols):

would be more readable